### DISCUSSION TITLE
Manual for version 4

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Manual#490] [General]
Hey Friends,

It's roughly six weeks, and then we will see Blender and Bforartists version 4. So not this much time anymore.

Version 4 means our manual needs changes in the header of every chapter.

And then there is the question if we shall migrate to a new, own repository with version 4 and start from scratch. Or shall we continue with the old repo? Version 3 has currently 1.7 gb in size. 1.52 Gb is git content. It's not this much really, but ...

Thoughts? Shall i make a task to change all chapters to version 4 already?

Kind regards
Reiner

**Comment by Draise14** [comment]
Keep going for now - change the headers then we branch like normal imo.

**Comment by ReinerBforartists** [comment]
Okay, so continue with the current repo. 

Shall we change the header already? Then i'll make a task.

**Comment by ReinerBforartists** [comment]
Done 

Change the header of all chapters to Bforartists version 4 #491
